nomos-core/chain-defs/src/tx/bundle.rs

```diff
@@ -2,55 +2,65 @@
 //crates
 use bytes::{Bytes, BytesMut};
 use risc0_zkvm::Prover;
-use serde::{de::Error as _, Deserialize, Deserializer, Serialize, Serializer};
 //internal
 use super::Error;
-use crate::wire;
-use nomos_proof_statements::bundle::{BundlePrivate, BundlePublic};
-
+use crate::{
+    proofs::{balance::BalanceProof, covenant::CovenantProof, ptx::PtxProof},
+    wire,
+};
 #[derive(Debug, Clone)]
 pub struct Bundle {
+    cm_roots: Vec<[u8; 32]>,
```

Comment:
Nitpick: Better explicit naming here. Not sure what is `cm` here atm 😅

Reply:
I would prefer to. I know its picky, but still 😅

Reply:
>  it's used fairly extensively in code/pseudocode 

I meant to say it's a common name used in literature and code by other projects such as zerocash/zcash, we were not the ones to introduce this abbreviation.
But again, we can use `committment`

Reply:
Ok, you know better. So lets go for the common literature then. Thanks!